n2y/plugins/linktopage.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from n2y.blocks import Block
+
+logger = logging.getLogger(__name__)
+
+class LinkToPage(Block):
+    """
+    Replace page link with the content of the linked-to page.
+
+    """
+    def __init__(self, client, notion_data, page, get_children=False):
+
+        super().__init__(client, page, notion_data, get_children)
+
+    def to_pandoc(self):
+        assert self.linked_page_id is not None
+
+        try:
+            self.client.get_page(self.linked_page_id) 
+            # Although out of the scope of the project at hand, get_page_or_database() 
+            # could be used in place of get_page, to cover the linked database case. 
+            # Might be an unnecessary step here. 
+            self.rich_text = self.client.wrap_notion_rich_text_array(self.notion_data["rich_text"], self)
+
+            # It's all about calling the proper method. 
+            content = self.rich_text.to_pandoc()
+
+        except PermissionError:
+            msg = 'Permission denied when attempting to access linked page having id [%s]'
+            logger.warning(msg, self.notion_data)
```

Comment:
I don't think `self.notion_data` is what you'd want to use here, since it will contain a lot more than the linked page id. Also, you'll want to be sure to include the URL to the LinkToPage block itself so that people who run into this error can quickly track down it's cause and fix it (by clicking on the URL and being taken to where the block is).